Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -66,18 +66,19 @@ class MultilineFormatter: PrettyFormatter {
     func objectString(
         typeName: String, fields: [(String, String)]
     ) -> String {
-        let prefix = "\(typeName)("
-        let body: String
-
         if fields.count == 1, let field = fields.first {
-            body = "\(field.0): \(field.1)"
+            return "\(typeName)(\(field.0): \(field.1))"
```

Comment:
Is the following easier to read?

```suggestion
            return "\(typeName)(" + "\(field.0): \(field.1)" + ")"
```